girder/api/rest.py

```diff
@@ -1193,3 +1193,9 @@ def wrapped(*args, **kwargs):
         return fun(ctx, *args, **kwargs)
 
     return wrapped
+
+class Prefix():
```

Comment:
use new-style class

Reply:
Sure, I copied ApiDocs which was old school, wan't sure why?

Reply:
oversight :) feel free to fix that too